src/modules/achievements/sagas.js

```diff
@@ -0,0 +1,135 @@
+import { all, call, fork, put, select, take, takeLatest } from 'redux-saga/effects'
+import ownT from './types'
+import ownA from './actions'
+import api from 'services/api'
+import stream from 'services/stream'
+import S from 'modules/selectors'
+import T from 'modules/types'
+import { eventChannel } from 'redux-saga'
+import blockchains from 'configurables/blockchains'
+
+const create = function * ({ link, title }) {
+  try {
+    const primaryBlockchainKey = blockchains.primary.key
+    const userAddress = yield select(S.wallets.primaryAddress)
+    yield call(api.createAchievement(primaryBlockchainKey), {
```

Comment:
Here we have a call to API that saves an achievement to cache layer. The goal of cache layer to reflect the actual state of blockchain. The requirement of cache layer is an opportunity to restore it from blockchain state at any moment. In order to fulfill this requirement, here client have to send transaction to loom network. It will implemented in next release.

Reply:
Can you create an Issue ticket in repo then?
Please detail it at best.
It's good to track ideas/things to implement in Issues so we dont forget them in PR and can come back to them later on.

Reply:
#31